src/SignalRServiceExtension/Client/IAzureSignalRSender.cs

```diff
@@ -10,5 +10,8 @@ internal interface IAzureSignalRSender
     {
         Task SendToAll(string hubName, SignalRData data);
         Task SendToUser(string hubName, string userId, SignalRData data);
+        Task SendToGroup(string hubName, string group, SignalRData data);
+        Task AddUser(string hubName, string userId, string group);
+        Task RemoveUser(string hubName, string userId, string group);
```

Comment:
```suggestion
        Task AddUserToGroup(string hubName, string userId, string groupName);
        Task RemoveUserFromGroup(string hubName, string userId, string groupName);
```

Consider more descriptive method names. Also `group` -> `groupName` to match other places.